### ISSUE TITLE
Add the complete feature into the Scratchpad

### ISSUE THREAD

**Issue by BrunoGeorgevich** [nbonamy/witsy#22]
I love using Witsy for my daily tasks! 

The Witsy scratchpad would be better with auto-completion.

Currently, you have to type "complete" in the command bar.

Adding custom features to the scratchpad would be great.

Thank you for creating this amazing tool!

**Comment by nbonamy** [comment]
Hey thanks for the positive feedback!

What do you mean exactly by:

1. Auto-completion: you mean just starting a text and ask to complete it? Is it not "Expand my writing" in the action bar?
2. Custom features: you mean custom actions in the action bar?

Thanks!

**Comment by BrunoGeorgevich** [comment]
> 1. Auto-completion: you mean just starting a text and ask to complete it? Is it not "Expand my writing" in the action bar?
Exactly. To complete only the sentence, for example. The expand feature generates more content than expected in the complete one.

> 2. Custom features: you mean custom actions in the action bar?
Yes, predefined prompts that will execute over the text.

**Comment by nbonamy** [comment]
Will be included in next version
